enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -1080,7 +1054,7 @@ for ReferenceGrants mitigates this risk.
 
 In the future, ReferenceGrant will likely be migrated out of Gateway API and into
 Kubernetes upstream. Until then, we will support ReferenceGrant as a part of
-Gateway API v1beta1 when using OSSM 2.4.
+Gateway API v1beta1 when using OSSM 3.x.
```

Comment:
```suggestion
Gateway API v1 when using OSSM 3.x.
```

Reply:
Well interestingly, it's not v1 (it's being held back because of the expectation of being available in core at some point), so I went for removing the version callout entirely for now.